README.md

```diff
@@ -2,92 +2,133 @@
 
-The network will be dependent on receiving new and updated state for new blocks. Full "bridge" nodes acting as benevolent state providers would be responsible for bringing in this data from the main network. The network should be able to remain healthy even with a small number of bridge nodes.
+The network will be dependent on receiving new and updated state for new blocks. Full "bridge" nodes acting as benevolent state providers would be responsible for bringing in this data from the main network. The network should be able to remain healthy even with a small number of bridge nodes.  As new data enters the network, nodes must be able to validate the data using a recent header from the header chain.
 
 Querying and reading data from the network should be fast enough for human-driven wallet operations, like estimating the gas for a transaction or reading state from a contract.
 
-### Chain History: Headers, Blocks, and Receipts
 
-In order to validate requested portal network data, a portal client needs to be able to request headers and validate their inclusion in the canonical header chain. Clients can use validated headers to further validate blocks, uncles, transactions, receipts, and state nodes.
+### History Network: Headers, Blocks, and Receipts
+
+The History Network facilitates on-demand retrieval of the history of the Ethereum chain.  This includes:
+
+- Headers
+- Block bodies
+- Receipts
 
-Normal clients will download every block header to construct the canonical chain. This is unreasonable for a "stateless" client. The "[double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571)" is the mechanism that enables portal clients to achieve this goal, without requiring them to download every canonical header.
+The responsibility for storing this data should be evenly distributed across the nodes in the network.  Nodes must be able to choose how much history data they want to store.  The data is distributed in a manner that allows nodes to determine the appropriate nodes to query for any individual piece of history data.  
 
-When a portal client requests a certain header, the response includes two accumulator proofs. As long as the client maintains an up-to-date view of the chain tip (via the gossip network) it can use the proofs to validate a headers inclusion in the canonical chain. Then, the client can use the header to validate other data retrieved from the portal network.
+Participants to this network are assumed to have access to the canonical header chain.
```

Comment:
s/to/in

This reads as if nodes must have independent access to the header chain (by connecting to a full node?) in order to participate in the history network.

Reply:
The plan is for them to use the "Header Gossip Network" but I'm trying not to anchor them to each other because they could just use Infura or a Full Node if they really want....